client/components/destination/index.js

```diff
@@ -109,13 +110,11 @@ const WeatherData = ({ geoJSON }) => {
         }
 
         if (lat && lon) {
-          setLatLng({ lat, lon });
           try {
-            // console.log(lat, lon)
-            // const weatherObjRes = await dispatch(fetchWeather({ lat, lon }));
-            // const weatherWkRes = await dispatch(fetchWeatherWeek({ lat, lon }));
-            // dispatch(setWeatherObject(weatherObjRes.payload));
-            // dispatch(setWeatherWeek(weatherWkRes.payload.list.slice(0, 4)));
+            const weatherObjRes = await dispatch(fetchWeather({ lat, lon }));
```

Comment:
This and the change above is what i was referring to

Reply:
Yes that's correct. I'm just referring to the files changed in this PR, it looks like there are modifications to the weather logic. I know that this PR is just for destination. 

If you look at the diff in files changed on GitHub, that's what I'm referring to.

I think it was unintentional change just due to force push previously, branches are likely out of sync. I can revert just wanted to confirm if that change was intended to be included in this PR. All good.

Reply:
Yeah. I didn't make those changes.